src/core/symbology-ng/qgsellipsesymbollayerv2.cpp

```diff
@@ -490,6 +490,10 @@ QgsSymbolLayerV2* QgsEllipseSymbolLayerV2::createFromSld( QDomElement &element )
   if ( !QgsSymbolLayerV2Utils::wellKnownMarkerFromSld( graphicElem, name, fillColor, borderColor, borderStyle, borderWidth, size ) )
     return nullptr;
 
+  QString uom = element.attribute( QString( "uom" ), "" );
```

Comment:
Looking back, *uom* could be a `const QString` here... It's also true for the others.

Reply:
tnx